backend.native/tests/interop/basics/ctoKString.def

```diff
@@ -0,0 +1,5 @@
+---
+const char* empty() { return ""; }
+const char* foo() { return "foo"; }
+const char* kuku() { return "куку"; }
+const char* invalid_utf8() { return "\x85\xAF"; }
```

Comment:
Maybe also a test with a string that has a `\0` in the middle?

Reply:
Nice idea, thanks! Done.